ci/perf-tester/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "build": "microbundle -f cjs --define 'navigator={}' --compress --no-sourcemap --target node src/index.js",
```

Comment:
Sorry, I'm not sure why we need to bundle sources. Could you tell me the reason?

Reply:
It was what the project I copied this over from did, but I think it’s OK to remove! Let’s find out ⬇️